app/views/papers/_form.html.erb

```diff
@@ -16,13 +16,20 @@
     <%= f.text_field :title %>
   </div>
 
-  <%= f.fields_for :authors, paper.authors do |a| %>
+  <%= f.fields_for :authors, (paper.authors.empty? ? 5.times { paper.authors.build } : paper.authors) do |a| %>
```

Comment:
We probably want to show `no. of authors + 1`, right?

Also, `5.times { paper.authors.build }` evaluates to `#=> 5`, and I'm not sure how `fields_for` handles an integer as an input.